tests/queries/0_stateless/03272_partition_pruning_monotonic_func_bug.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE tt
```

Comment:
```suggestion
DROP TABLE IF EXISTS tt;

CREATE TABLE tt
```

Reply:
Let's add DROP at start and end.

Reply:
I don't understand why we need it; it only makes the test longer.

Reply:
Because some people, mostly Alexey, run the sql tests directly via clickhouse-client instead of using `clickhouse-test`.